db/migrate/20151014135153_add_redirects_for_raib.rb

```diff
@@ -0,0 +1,20 @@
+class AddRedirectsForRaib < Mongoid::Migration
+  def self.up
+    existing_redirects = CSV.read("#{Rails.root}/db/migrate/files/raib_redirects.csv")
+
+    existing_redirects.each do |from_base_path, to_base_path|
+      content_item = ContentItem.find_by(base_path: from_base_path)
+      content_item.update_attributes!(
+        content_id: SecureRandom.uuid,
```

Comment:
I'm not keen on assigning random values from a migration - it means that if the migration is run multiple times, we'll get different results.  I'd be happier with those random values going into the CSV file that's added in this commit.